mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -546,7 +554,12 @@ def __call__(self, datasets):
         # alignment, i.e. the SL ROIs cover roughly the same area
         queryengines = self._get_trained_queryengines(
             datasets, params.queryengine, params.radius, params.ref_ds)
-
+        # For surface nodes to voxels queryengines, roi_seed hardly makes sense
+        if isinstance(queryengines[params.ref_ds], SurfaceVerticesQueryEngine):
+            self.force_roi_seed = False
```

Comment:
That will be useful, not only for this but also to fix how `roi_seed` can be wrong for `Searchlight` with `SurfaceVerticesQueryEngine` in the case when `node_id` happens to match with `feature_id`. I will leave it upto the masters to make that decision ;)